components/serverless/config/crd/bases/serverless.kyma-project.io_functions.yaml

```diff
@@ -185,6 +185,9 @@ spec:
                 description: Defines labels used in Deployment's PodTemplate and applied
                   on the Function's runtime Pod.
                 type: object
+                x-kubernetes-validations:
+                - message: Labels has key starting with serverless.kyma-project.io/
+                  rule: '!(self.exists(e, e.matches(''serverless.kyma-project.io/*'')))'
```

Comment:
We should escape `.` and `*` inside regex pattern.
`serverless.kyma-project.io/*` -> `serverless\.kyma-project\.io/\*`